frontend/public/components/storage-class-form.tsx

```diff
@@ -1000,6 +1020,22 @@ export class StorageClassForm_ extends React.Component<
             </span>
           </div>
 
+          {expansionFlag && (
+            <div className="checkbox">
+              <label>
+                <input
+                  type="checkbox"
+                  className="create-storage-class-form__checkbox"
+                  onChange={(event) => this.setStorageHandler('expansion', event)}
+                  checked={isExpansion}
+                  id="expansion-checkbox"
+                  aria-describedby="expansion-checkbox"
```

Comment:
It doesn't make sense to have `aria-describedby` point back to the input `id`. Just remove these two lines. Since the `label` wraps the `input`, they're not needed.